ares/ps1/peripheral/port.cpp

```diff
@@ -12,6 +12,7 @@ auto PeripheralPort::load(Node::Object parent) -> void {
   port->setType(type);
   port->setHotSwappable(true);
   port->setAllocate([&](auto name) { return allocate(name); });
+  port->setDisconnect([&] { device.reset(); });
```

Comment:
I did not draw a distinction here for controllers vs. memory cards, though I could have. As with the NG cards, there is currently no associated UI.